aten/src/THCUNN/VolumetricUpSamplingTrilinear.cu

```diff
@@ -7,7 +7,7 @@
 #include "THCDeviceTensor.cuh"
 #include "THCDeviceTensorUtils.cuh"
 #include "THCDeviceUtils.cuh"
-#include "THCHalf.h"
+#include "TH/THHalf.h"
```

Comment:
why are you including TH/THHalf.h all over the place?  That's only needed for C compatibility, right?  So anything that's not a .h should just include ATen/Half.